tools/hyphy/compile.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
Hmm, this gives me errors like:

```
/src/biowasm/tools/hyphy/src/src/core/matrix.cpp:3431:50: error: use of undeclared identifier 'vld1q_lane_f64'
                                                 vld1q_lane_f64 (storage.theData + s0, R1, 0), 1);
                                                 ^
...
```

Curious which version of Emscripten you're using?

I'm still on 2.0.25 (https://github.com/biowasm/biowasm/blob/main/CONTRIBUTING.md#development-setup)

Reply:
I had to make an edit to our CMake configuration in order for this to work. Let me see if I can set a flag.

Reply:
Support has been added in HyPhy 2.5.3